gc/verbose/handler_standard/VerboseHandlerOutputStandard.cpp

```diff
@@ -26,6 +26,7 @@
 #include "gcutils.h"
 
 #include "ConcurrentGCStats.hpp"
```

Comment:
We don't need this line any more since it included in `ConcurrentMarkPhaseStats`. Not sure we could delete it.

Reply:
It is doable. "ConcurrentGCStats.hpp" is included in "ConcurrentMarkPhaseStats.hpp", but not sure it is a good practice though.

Reply:
Don't rely on headers being included transitively. You should leave it, a reason being that it will cause issues if we refactor or makes changes down the road to `ConcurrentMarkPhaseStats.hpp`